manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -586,16 +586,6 @@ spec:
           verbs:
           - get
           - list
-        - apiGroups: ["iter8.tools"]
-          resources:
-          - experiments
-          verbs:
-          - get
-          - list
-          - watch
-          - create
-          - delete
-          - patch
```

Comment:
We probably need to revert this. This `manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml` is the CSV for the productized Kiali operator. This must support v1.12 and v1.24 as well as v1.36 Kiali servers. Because I assume those older versions of Kiali still support the iter8 extension, we need our productized CSV to keep this permission (otherwise the older productized versions of Kiali server will not be able to be installed by the productized operator).

We can remove this section only when all supported versions of Kiali server no longer support the iter8 extension.

Reply:
Reverted.
Probably the risk of problems is very low as this is unused in the product but better to be conservative here.